WebKitBrowser/WebKitImplementation.cpp

```diff
@@ -3190,6 +3193,9 @@ static GSourceFuncs _handlerIntervention =
 
             auto preferences = WKPreferencesCreate();
 
+            // ICE candidate filtering
+            webkit_settings_set_enable_ice_candidate_filtering(preferences, _config.ICECandidateFilteringEnabled.Value());
```

Comment:
> It doesn't make sense to use glib API with WK preferences. Perhaps merging problem?
Indeed, that was a merging problem. My original change was placed just after webaudio settings in GLIB part, but after merging to rdkservices upstream I didn't notice that this line was applied in the wrong place.

> I'd prefer to avoid binary dependency on webkit_settings_set_enable_ice_candidate_filtering. Please change to use object property setter instead.
Agreed and fixed.